dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/events/EventExportTests.java

```diff
@@ -141,9 +141,9 @@ public void beforeEach()
     Stream<Arguments> shouldUseCorrectScopeWhenOuIsProvided()
     {
         return Stream.of(
-            Arguments.of( "OU: root", "SELECTED", rootOu, false, null ),
-            Arguments.of( "OU: capture", "SELECTED", captureOu, true, Arrays.asList( captureOu ) ),
-            Arguments.of( "OU: search", "SELECTED", searchOu, false, null ),
+            Arguments.of( "OU: root", "selected", rootOu, false, null ),
```

Comment:
That was really just to test. I am looking for a better place in web-api-test to add a relevant test for it